src/PKSim.Infrastructure/Serialization/ORM/Mappers/ProjectToProjectMetaDataMapper.cs

```diff
@@ -33,7 +33,8 @@ public class ProjectToProjectMetaDataMapper : IProjectToProjectMetaDataMapper,
       IVisitor<Formulation>,
       IVisitor<RandomPopulation>,
       IVisitor<PKSimEvent>,
-      IVisitor<ImportPopulation>
+      IVisitor<ImportPopulation>,
+      IVisitor<PKSimObserverBuildingBlock>
```

Comment:
I could rename PKSimObserverBuildingBlock to ObserverSet? Maybe it's easier to read?

Reply:
@Yuri05 yeah I think renaming the object PKSimObserverBuildingBlock to ObserverSet will be much clearer. Otherwise we can confuse it with the one from core or it's just too long

Reply:
Not sure why we would PKSimObserverBuildingBlock with ObserverBuildingBlock from core, but also fine with ObserverSet

Reply:
the problem is that we never use the term building block anywhere. We use Individual and not IndividualBuildingBlock. Also our observer are always singular by construction . Using another name is simply more consistent

Reply:
ok